docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -353,14 +353,40 @@ This value is `true` by default.
 
 ### ttl
 
-_Time to live_ (TTL) is the time that a query result is stored in the caching system before it is deleted or refreshed. This setting defines the time to live for query caching, when TTL is not configured in data source settings. The default value is `5m` (5 minutes).
+_Time to live_ (TTL) is the time that a query result is stored in the caching system before it is deleted or refreshed. This setting defines the time to live for query caching, when TTL is not configured in data source settings. The default value is `1m` (1 minute).
+
+### max_ttl_ms
```

Comment:
@osg-grafana i'm going to update the setting parsing logic to match how `ttl` handles durations. i'll leave this PR in draft and update the docs once the related change merges